pointgrey_camera_driver/src/nodelet.cpp

```diff
@@ -160,6 +160,18 @@ class PointGreyCameraNodelet: public nodelet::Nodelet
   void connectCb()
   {
     NODELET_DEBUG("Connect callback!");
+
+    // Waiting for exposure node (a prerequisite) to finish being started for cam8
+    if (frame_id_ == "cam8")
+    {
+      ROS_INFO_STREAM("Waiting for cam8 exposure node to finish starting...");
+      const bool ret = ros::service::waitForService("/dynamic_exposure/"+frame_id_+"/ready", 20000); // 20 seconds
```

Comment:
Nit: Prefer ros::Duration(20); which is unit-safe and makes it obvious what the units are

Reply:
using ros::Duration!